eva-accession-release-automation/publish_release_to_ftp/publish_release_files_to_ftp.py

```diff
@@ -96,9 +96,9 @@ def get_release_file_list_for_assembly(release_assembly_info):
     for example, see here, ftp://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_1/by_assembly/GCA_000001515.4/)
     """
     assembly_accession = release_assembly_info["assembly_accession"]
-    vcf_files = ["{0}_{1}.vcf.gz".format(assembly_accession, category) for category in release_vcf_file_categories]
-    text_files = ["{0}_{1}.txt.gz".format(assembly_accession, category) for category in release_text_file_categories]
-    csi_files = ["{0}.csi".format(filename) for filename in vcf_files]
+    vcf_files = [f"{assembly_accession}_{category}.vcf.gz" for category in release_vcf_file_categories]
```

Comment:
These filenames will include the taxonomy id as well - I see it included in your test run but I don't see where it's added in the code, maybe I've missed something.

Reply:
sorry, this was part of the 2nd commit which I forgot to push. 
Have pushed now